<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordDialog</class>
 <widget class="QDialog" name="RecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="yspcclient.qrc">
    <normaloff>:/res/login/logo.png</normaloff>:/res/login/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#RecordDialog
{
background-color: rgb(255, 255, 255);
border-left:1px solid rgb(36,36,36);
border-top:0px;
border-right:1px solid rgb(36,36,36);
border-bottom:1px solid rgb(36,36,36);
}</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;宋体&quot;;
color: rgb(179,179,179);</string>
   </property>
   <property name="text">
    <string>(限200字)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>10</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>发布录像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>12</y>
      <width>23</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 20px &quot;宋体&quot;;
color: rgb(255, 255, 255);
border:none;

QPushButton::disabled
{
color: rgb(229, 229, 229);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>65</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>课件名称：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editCoursewareNote">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>110</y>
     <width>359</width>
     <height>119</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(179,179,179);
font-size:12px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>149</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>课件要点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCoursewareName">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>60</y>
     <width>359</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(179,179,179);
font-size:12px;</string>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpload">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>250</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: transparent;
	font: 14px &quot;宋体&quot;;
	border:1px solid rgb(0,181,255);
	border-radius:3px;
	color: rgb(0, 181, 255);
}
QPushButton::disabled
{
	background-color: rgb(149, 149, 149);
}
</string>
   </property>
   <property name="text">
    <string>确认发布</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>250</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 14px &quot;宋体&quot;;
border:1px solid rgb(179,179,179);
border-radius:3px;
color: rgb(179, 179, 179);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>85</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;宋体&quot;;
color:rgb(179,179,179);</string>
   </property>
   <property name="text">
    <string>(限20字)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="yspcclient.qrc"/>
 </resources>
 <connections/>
</ui>
